sktime/forecasting/model_selection/tests/test_tune.py

```diff
@@ -348,3 +351,22 @@ def test_gscv_backends(backend_set):
         backend_params=backend_params,
     )
     gscv.fit(y, X)
+
+
+@pytest.mark.parametrize("return_n_best_forecasters", [-1, 0, 1, 2])
```

Comment:
`Grid` search is easy to test because its behaviour is deterministic since it tries every possible combination, so I can know exactly how many models/forecasts will be generated even before it runs, while `Random` and `Skopt` search use some kind of heuristic (idk what) to optimise the search, so I'm not sure how to test them.